Week3/app.js

```diff
@@ -0,0 +1,239 @@
+{
+  /*
+   * Q1.
+   * Write a program which would sort the below array in ascending order by name.
+   * Hint: String.prototype.localeCompare()
+   */
+  const people = [
+    {
+      name: 'Richie Baros',
+    },
+    {
+      name: 'Jayson Karner',
+    },
+    {
+      name: 'Dylan Wanner',
+    },
+    {
+      name: 'Travis Tarwater',
+    },
+    {
+      name: 'Angelique Lyall',
+    },
+    {
+      name: 'Stacie Dyar',
+    },
+  ];
+
+  people.sort(function(a, b) {
+    return a.name.localeCompare(b.name);
+  });
+  console.log(people);
```

Comment:
nice! I'm not sure if this was part of the homework, but because you submitted it I will check it :)

Reply:
this part from Yash. He asks to us make solution this question